spec/features/system_admin/upload_funding/viewing_upload_funding_spec.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+feature "Viewing upload funding" do
+  before do
+    given_i_am_authenticated(user:)
+    when_i_visit_the_root_page
+    and_i_click_on_support_for_register
+  end
+
+  context "user is a system admin" do
+    let(:user) { create(:user, :system_admin) }
```

Comment:
I think we need to settle on whether we're using `let`s or ivars as originally decided for feature specs to be consistent

Reply:
Fixed, old habits